config/bspo.yml

```diff
@@ -6,6 +6,7 @@ base_url: /obo/bspo
 products:
 - bspo.owl: https://raw.githubusercontent.com/obophenotype/biological-spatial-ontology/master/bspo.owl
 - bspo.obo: https://raw.githubusercontent.com/obophenotype/biological-spatial-ontology/master/bspo.obo
+- bspo.json: https://raw.githubusercontent.com/obophenotype/biological-spatial-ontology/master/bspo.json
```

Comment:
This looks correct, but I get a 404. Should I go ahead anyway?

Reply:
Oh sorry, fixed in https://github.com/obophenotype/biological-spatial-ontology/pull/31

Reply:
Fixed now, thank you!